Makefile

```diff
@@ -501,11 +501,14 @@ fmt: tools ## Run goimports
 vet: ## Run go vet
 	$(GO) vet ./...
 
-check: misspell lint
+check: misspell yamllint lint 
```

Comment:
@aman556 heads up for the change proposed for this line https://github.com/vmware-tanzu/tanzu-framework/pull/1450/files

Reply:
Thanks @navidshaikh will do this change.